sc/src/main/javascript/build-bundles.sh

```diff
@@ -18,9 +18,13 @@ do
     pushd . > /dev/null
     echo "Upgrading $dir:"
     cd $dir
-    yarn upgrade
+    #yarn upgrade
+    ncu -a
```

Comment:
which ncu? I don't have that

Reply:
I added a note in the commit message, just do `npm install -g npm-check-updates`. It’s quite useful. In my mind it’s what I would like `npm update`/`yarn upgrade` to do.